config/routes.rb

```diff
@@ -1,9 +1,8 @@
 Rails.application.routes.draw do
 
-  resources :websites, as: nil
+  resources :websites, path: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.